### PR TITLE
Rollback #1759

### PR DESCRIPTION
**Pull request by riemannulus**
Since #1759, we had a problem with preload. So I was reverted.

If you look at the following code, there is a part where you made a mistake while deleting the parameter. 

https://github.com/planetarium/libplanet/pull/1792/files#diff-215bb45126357b0974dea14401f0213b497011890a129ae3a7df3e3468ea4911L524

If preload is always true, this `if` statement is meanless but I was left this code blocks.

In this PR. I'm re-implement #1759 and fixing those bugs.

### CODE REVIEW

**Comment by dahlia** [comment]
@riemannulus Could you elaborate some context/background behind this patch in the pull request description?

File: CHANGES.md [hunk 1]
```diff
@@ -32,6 +32,11 @@ To be released.
 
 ### Behavioral changes
 
+  - (Libplanet.Net) Return the fixed races between `Swarm<T>`'s dual systems
```
Comment (longfin) [inline CHANGES.md:35]: It seems the link for #1789 was missing.

File: Libplanet.Net/Swarm.BlockSync.cs [hunk 2]
```diff
@@ -74,28 +74,123 @@ public partial class Swarm<T>
                 return;
             }
 
+            var totalBlocksToDownload = 0L;
+            Block<T> tempTip = BlockChain.Tip;
+            List<Block<T>> blocks = new List<Block<T>>();
```
Comment (longfin) [inline Libplanet.Net/Swarm.BlockSync.cs:79]: Trivial

```suggestion
            var blocks = new List<Block<T>>();
```

**Review by longfin — COMMENTED** [review]
Does it resolve the root cause of #1759 and #1792? if so, could you leave the comments about what was the matter?

File: Libplanet.Net/Swarm.BlockSync.cs (hunk 2 above)
Comment (riemannulus) [reply Libplanet.Net/Swarm.BlockSync.cs]: https://github.com/planetarium/libplanet/pull/1798/commits/b7a1d200156030175ad3ea75231b09977f74cc5a amended

File: CHANGES.md (hunk 1 above)
Comment (riemannulus) [reply CHANGES.md]: https://github.com/planetarium/libplanet/pull/1798/commits/f8775fc1d15efe1a0453adfb97561d185f466098

**Review by dahlia — APPROVED** [review]

**Pull request merged**